src/test/regress/expected/with_basics.out

```diff
@@ -0,0 +1,274 @@
+-- CTE cannot be outside of FROM/WHERE clause
+WITH cte AS (
+	SELECT user_id FROM users_table WHERE value_2 IN (1, 2)
+)
+SELECT (SELECT * FROM cte);
+WARNING:  more than one row returned by a subquery used as an expression
+CONTEXT:  while executing command on localhost:57638
+ERROR:  could not receive query results
+WITH cte_basic AS (
+	SELECT * FROM users_table
+)
+SELECT (SELECT user_id FROM cte_basic), user_id FROM users_table;
+ERROR:  could not run distributed query with subquery outside the FROM and WHERE clauses
+HINT:  Consider using an equality filter on the distributed table's partition column.
+WITH cte_1 AS (
```

Comment:
We could skip the tests with Postgres errors. They might be confusing to the readers.